### DISCUSSION TITLE
Setting multiple tags in rules action Add tag?

### DISCUSSION THREAD

**Discussion by thehijacker** [firefly-iii/firefly-iii#9034] [Firefly III] [answered]
I have a rule that will set transaction category and tag. Now I wish to add two tags to it. I tried setting two Add tag actions but only one is made. I tried to input string as "Tag1, Tag2" or "Tag1; Tag2" but it sitll only makes one. What would be the proper syntax?

Thank you.

**Comment by JC5** [comment, marked as answer]
Add the action twice.
